Mathlib/NumberTheory/ModularForms/EisensteinSeries/ModularForm.lean

```diff
@@ -0,0 +1,30 @@
+/-
+Copyright (c) 2024 Chris Birkbeck. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Birkbeck
+-/
+
+import Mathlib.NumberTheory.ModularForms.Basic
+import Mathlib.NumberTheory.ModularForms.EisensteinSeries.IsBoundedAtImInfty
+import Mathlib.NumberTheory.ModularForms.EisensteinSeries.MDifferentiable
+
+/-!
+# Eisenstein series are Modular Forms
+
+We show that Eisenstein series of weight `k` and level `Γ(N)` with congruence condition
+`a : Fin 2 → ZMod N` are Modular Forms.
+-/
+
+noncomputable section
+
+
+namespace EisensteinSeries
+
+/-- This defines Eisenstein series as modular forms of weight `k`, level `Γ(N)` and congruence
+condition given by `a: Fin 2 → ZMod N`. -/
+def eisensteinSeries_MF {k : ℤ} {N : ℕ+} (hk : 3 ≤ k) (a : Fin 2 → ZMod N) :
```

Comment:
The name `EisensteinSeries.eisensteinSeries_MF` seems weird. Maybe `ModularForms.eisensteinSeries`?

Reply:
Hmm good point, we actually already have `EinsesteinSeries.eisensteinSeries` which seems like a bad thing. I might put this in the root namespace and then make the other one `ModularForms.eisensteinSeries_MF` to denote its a mod form. Does that sound ok?

Reply:
Yes, no problem.